src/main/java/chat/ChatRequestModel.java

```diff
@@ -0,0 +1,64 @@
+package chat;
+
+import java.util.Date;
+
+public class ChatRequestModel {
+
+    private int chatId;
+
+    private String content;
+
+    private int author;
+
+    private int receiver;
+
+    private Date sendTime;
+
+    public ChatRequestModel(int chatId, String content, int author, int receiver, Date sendTime) {
```

Comment:
nit: for `ChatRequestModel` maybe worth to use a single pair of getters and setters?